jest.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     '^uniforms([^/]*)(.*)$': '<rootDir>/packages/uniforms$1/src$2',
   },
   preset: 'ts-jest',
-  setupFiles: ['./scripts/setupEnzyme.ts', './scripts/setupFilterWarnings.ts'],
+  setupFiles: ['./scripts/setupRTS.ts', './scripts/setupFilterWarnings.ts'],
```

Comment:
What is `RTS`? That file looks like configuration needed for the `JSDOM` ([docs](https://jestjs.io/docs/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom)). Maybe use `JSDOM` in the filename.

Reply:
RTS = React Testing Library

Reply:
I would say `RTL` is better than `RTS`, but still it isn't setting up React Testing Library, but Jest.

Reply:
i change name for just` setupJest`